base_layer/core/src/base_node/sync/config.rs

```diff
@@ -46,7 +46,7 @@ pub struct BlockchainSyncConfig {
 impl Default for BlockchainSyncConfig {
     fn default() -> Self {
         Self {
-            initial_max_sync_latency: Duration::from_secs(3),
+            initial_max_sync_latency: Duration::from_secs(4),
```

Comment:
Let's bump this to 10 for now
```suggestion
            initial_max_sync_latency: Duration::from_secs(10),
```